### ISSUE TITLE
FileNotFoundError on 2_download_data_train.py

### ISSUE THREAD

**Issue by yingjin-us** [Azure/cortana-intelligence-product-detection-from-images#4]
Tried following technical deployment to test the model, on step covering download training data, I ran into an error:
Traceback (most recent call last):
  File "2_download_data_train.py", line 72, in <module>
    with open(fname_box, 'w') as f_box, \
FileNotFoundError: [Errno 2] No such file or directory: 'technical_deployment\\data_management\\..\\train_model\\data\\grocery\\negative\\d39fd583-8e4b-4c59-b12e-c4f00acd8584.bboxes.tsv'

After looking at 2_download_data_train.py, I think the call to file open is not correct.

**Comment by lixzhang** [comment]
Can you provide more details on the error message? The file "2_download_data_train.py" is used to download annotations from DocDB and images from Azure Blob, not from a local path.

**Comment by yingjin-us** [comment]
I was using Windows DLVM on Azure to download training data and train the model. And as the error message shown, the traceback happened at line 72 of the 2_download_data_train.py. That line was to open files and  write to them the rois.

**Comment by lixzhang** [comment]
Did you follow the instructions to create DocDB account and Azure Storage account first? After you've done that, you can update config.py and run 1_upload_historical_data.py to upload data. Only after these steps can 2_download_data_train.py be run.

**Comment by yingjin-us** [comment]
Yes, I did follow the instructions. DocDB and storage account are all
created. That is why in the error message the not-found filename has an
image id in the name.
I tried changing the call to open(fname_box,'w+') (create file if not
exist), but still got the same error.

**Comment by yingjin-us** [comment]
Hi, Lixin,
With some trial and errors, I figured it out why the error happens. The error is caused by path notation difference between windows (ex. C:\root\subdir) vs python way of path notation (ex. C:/root/subdir). I made some changes in config.py under data-management folder to make python happy about path to various folders and the errors went away. 
I will make it do for now, and will come back and make a more general fix and will generate a PR.
For now I will close this issue.
Thanks
